Instanssi/main2020/templates/main2020/english.html

```diff
@@ -16,40 +16,88 @@ <h1>In English</h1>
-<p>
-    The address of the venue is Voionmaankatu 17-19, 40700 Jyväskylä, Finland.
+    Competitions at Instanssi include classics such as demo (several categories), music, graphics, and Summamutikka, a free-form compo, which is meant to confuse the audience using any (legal) means necessary. Another speciality of Instanssi are programmable lights, which can be used as part of compo entries.
 </p>
 
 
+<h2>Date and venue</h2>
+    <p>
+      Instanssi 2020 will be organized on 27.2.-1.3.2020 at Jyväskylän kristillinen koulu (address: Voionmaankatu 18, Jyväskylä).
+    </p>
+
+    <iframe width="600"
+            height="450"
+            id="instanssi-location-map"
+            frameborder="0"
+            style="border:0; max-width:100%"
+            src="https://www.google.com/maps/embed/v1/place?q=place_id:ChIJ3WoIEiN0hUYRogdPWJx7Alk&language=fi&key={{ GOOGLE_API_KEY }}" allowfullscreen></iframe>
+
+<h2>Contact</h2>
+    <p>
+        IRC: #instanssi @ IRCNet<br />
+        Telegram: <a href="https://telegram.me/Instanssi">https://telegram.me/Instanssi</a><br /> <br />
+        Email: You can contact the organizers via info@instanssi.org.<br />
+    </p>
+
 <h2>Tickets</h2>
 <p>
-    Tickets can be bought from the door or beforehand from the Internet. From the door you can also get
-    a ticket, which gives you one time entrance and costs 2 Euros
+    Tickets can be bought from the door or beforehand from the <a href="https://instanssi.org/store">Instanssi store</a> on our website. 
 </p>
-<p
-    <b>Dynamic ticket</b> – 5 Euros. This ticket includes entrance to the event for the whole weekend.
-    You can watch all the program, participate in the workshops and use the showers and WLAN. There is
-    also sleeping space available.
-</p>
-<p>
-    <b>Static ticket</b> – 10 Euros. This ticket includes everything that the Dynamic ticket offers,
-    plus network, electricity and table space (90x70cm) for your computer. Note: static tickets have
-    been sold out.
-</p>-->
+<ul>
+    <li>
+        <b>Entrance ticket (kävijälippu)</b> – 5 Euros. This ticket includes entrance to the event for the whole weekend. You can enjoy the program, participate in the workshops, and stay the night.
+    </li>
+    <li>
+        <b>Early Bird Computer place ticket (konepaikkalippu)</b> – 20 Euros. This ticket allows entrance to Instanssi and a table spot for your computer.
+    </li>
+    <li>
+        <b>Computer place ticket (konepaikkalippu)</b> – 25 Euros. This ticket allows entrance to Instanssi and a table spot for your computer.
+    </li>
+</ul>
+
+
+<h2>What do I need to take with me?</h2>
+    <p>
+        Bring the same necessities as you would to any other LAN-party or demoparty. Here is a small checklist to help you.
+    </p>
+    <ul>
+      <li>
+          If you have a computer place ticket, bring your computer with all the necessary accessories. Bring a long power cord and your own extension cord. Remember to write your name on your stuff!
```

Comment:
Should this be "If you have a ticket with a computer place" so that it refers to all ticket types that have room for a computer ?